codeviz.py

```diff
@@ -29,16 +29,16 @@ def __init__(self, filename):
     def get_includes(self, filename):
         includes_re = re.compile(r'\s*#\s*include\s+["<](?P<file>.+?)[">]')
 
-        with open(filename, 'rt') as f:
-            data = f.read()
-
-        # Remove all comments
-        data = re.sub(r'/\*.*?\*/', '', data, flags=re.DOTALL)
-        data = re.sub(r'//.*', '', data)
-
-        includes = includes_re.findall(data)
-
-        return includes
+        try:
+            with open(filename, 'rt') as f:
+                    data = f.read()
+            # Remove all comments
+            data = re.sub(r'/\*.*?\*/', '', data, flags=re.DOTALL)
+            data = re.sub(r'//.*', '', data)
+            return includes_re.findall(data)
+        except UnicodeError as ex:
+                print(f"Cann't read file: \"{filename}\", position: {ex.start}")
```

Comment:
Typo ("Cann't"), also use single quote strings. Best not to use contractions. Instead, "Cannot read file".